sdk/lib/cicero/CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+
+list(APPEND SOURCE
+    cicarray.cpp
+    cicbase.cpp
+    ciccaret.cpp
+    cicfmap.cpp
+    cicimc.cpp
+    cicreg.cpp
+    cicuif.cpp)
+
+add_library(cicero STATIC ${SOURCE})
+target_compile_definitions(cicero PRIVATE UNICODE _UNICODE)
+add_dependencies(cicero psdk)
+
+add_library(cicero_include INTERFACE)
```

Comment:
What is the actual purpose of this second library?
Aren't its contents already compiled within the main "cicero" library above?

Reply:
This `cicero_include` is an `INTERFACE` library, and will add the include directory.

Reply:
Is there any actual advantage of doing that, instead of doing what we use to do so far with all our other libraries, namely adding the path to where their headers reside in a (target_)include_directories cmake statement for the module that needs those?

Reply:
United `cicero_include` into `cicero`.

Reply:
Fixed.